lib/pushmi_pullyu/swift_depositer.rb

```diff
@@ -1,13 +1,11 @@
+require 'digest/md5'
 require 'openstack'
 
 class PushmiPullyu::SwiftDepositer
 
-  attr_reader :swift_connection, :swift_container
-
-  def initialize(connection, container)
-    raise 'connection can not be nil' if connection.nil?
-    raise 'container can not be nil' if container.nil?
```

Comment:
If container is nil, this will raise an expection when we attempt to use it down on line 24. Let ruby do its job and raise the proper exception instead of our custom one.